ipyvolume/pylab.py

```diff
@@ -425,9 +425,9 @@ def quiver(x, y, z, vx, vy, vz, size=default_size * 10,
     :param x: {x}
     :param y: {y}
     :param z: {z}
-    :param vx: {x_dir}
-    :param vy: {y_dir}
-    :param vz: {z_dir}
+    :param vx: x_dir
```

Comment:
u,v,w is more consistent with matlab https://nl.mathworks.com/help/matlab/ref/quiver3.html and matplotlib https://matplotlib.org/mpl_toolkits/mplot3d/tutorial.html#line-plots